themes/src/main/resources-community/theme/base/login/messages/messages_sk.properties

```diff
@@ -24,20 +31,45 @@ realmChoice=Realm
 unknownUser=Neznámy používateľ
 loginTotpTitle=Nastavenie mobilného autentifikátora
 loginProfileTitle=Aktualizácia informácií o účte
+loginIdpReviewProfileTitle=Aktualizácia informácií o účte
 loginTimeout=Prihlasovanie trvalo príliš dlho. Prihlasovací proces začína od začiatku.
-oauthGrantTitle=Poskytnúť prístup
+reauthenticate=Ak chcete pokračovať, znova sa overte
+oauthGrantTitle=Poskytnúť prístup {0}
 oauthGrantTitleHtml={0}
+oauthGrantInformation=Uistite sa, že dôverujete {0} tým, že sa dozviete, ako bude {0} spracovávať vaše údaje. 
+oauthGrantReview=Môžete si prezrieť 
+oauthGrantTos=podmienky služby.
+oauthGrantPolicy=zásady ochrany osobných údajov.
 errorTitle=Je nám ľúto ...
 errorTitleHtml=<strong>Ospravedlňujeme</strong> sa ...
 emailVerifyTitle=Overenie e-mailom
 emailForgotTitle=Zabudli ste heslo?
+updateEmailTitle=Aktualizovať e-mail 
+emailUpdateConfirmationSentTitle=Odoslaný potvrdzujúci e-mail
+emailUpdateConfirmationSent=Na adresu {0} bol odoslaný potvrdzujúci e-mail. Na dokončenie aktualizácie e-mailu musíte postupovať podľa pokynov prvého z nich.
+emailUpdatedTitle=Email aktualizovaný
+emailUpdated=Email účtu bol úspešne aktualizovaný na {0}.
 updatePasswordTitle=Aktualizácia hesla
 codeSuccessTitle=Kód úspechu
 codeErrorTitle=Kód chyby\: {0}
+displayUnsupported=Nepodporovaný typ displeja 
+browserRequired=Pre prihlásenie je potrebný prehliadač
+browserContinue=Pre dokončenie prihlásenia je potrebný prehliadač
+#browserContinuePrompt=Otvoriť prehliadač a pokračovať v prihlasovaní? [a/n]:
+#browserContinueAnswer=a
```

Comment:
Why is this commented?

Reply:
I wasn't sure how this works in the code. If it works like this, it can be uncommented